lib/launchdarkly_flutter.dart

```diff
@@ -3,6 +3,102 @@ import 'dart:async';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/services.dart';
 
+class LaunchDarklyConfig {
+  final bool allAttributesPrivate;
+  final Set<String> privateAttributes;
+
+  const LaunchDarklyConfig({
+    this.allAttributesPrivate = false,
+    this.privateAttributes = const {},
+  });
+
+  Map<String, dynamic> toMap() => {
+        'allAttributesPrivate': allAttributesPrivate,
+        'privateAttributes': privateAttributes.toList(),
+      };
+}
```

Comment:
Can we please have this class externalised to its own file (such as `lauchdarkly_config.dart`) and properly commented? Thanks :)